Changes for https support

Deadolus opened the pull request:
These are changes to support https

phkehl commented on tools/multipleJobsToTschenggins.sh:
```diff
@@ -29,7 +29,13 @@ function update {
-		http://$BACKENDUSER:$BACKENDPASS@$BACKENDSERVER:$BACKENDPORT/tschenggins-laempli/ng/ > /dev/null
+		https://$BACKENDUSER:$BACKENDPASS@$BACKENDSERVER:$BACKENDPORT/tschenggins-laempli/  \
+        --cacert tsueri.oo.crt --insecure \
+        -A "Tschenggins-Bash-Script" \
```
Maybe better -A "multipleJobsToTschenggins.sh/1.0" ?
To be in line with the other scripts and the typical agent string format.

phkehl commented on tools/multipleJobsToTschenggins.sh:
```diff
@@ -29,7 +29,13 @@ function update {
 		--data '{"cmd":"update","debug":0,"states":[{"name":"'$3'","result":"'$1'","server":"'$TARGETSERVER'","state":"'$2'"}]}' \
-		http://$BACKENDUSER:$BACKENDPASS@$BACKENDSERVER:$BACKENDPORT/tschenggins-laempli/ng/ > /dev/null
+		https://$BACKENDUSER:$BACKENDPASS@$BACKENDSERVER:$BACKENDPORT/tschenggins-laempli/  \
+        --cacert tsueri.oo.crt --insecure \
```
No leaks please! Remove any reference to our existing setup. Perhaps make it a command line option.

Deadolus commented:
I addressed your comments. You were of course right. 
I force pushed a rewritten history to delete some information from github. 
If you are ok with the changes, you may now merge this updated branch.

phkehl commented:
Merci!
